### DISCUSSION TITLE
How to build my website in real time.

### DISCUSSION THREAD

**Discussion by hansemso** [community/community#43846] [New to GitHub]
Hi.  I don't get how github desktop and visual studio code work.  Desktop gives me 3 choices, one of which is to use vsc.  So, I open my index.html file in it and make changes.  How do I push it to github using desktop?  I just want to be able to see my changes take effect in real time.  In other words, to make changes and then preview it.  I went to one free online course and it had me on a split screen so that when I made changes to the code I could see it on the right side.  That would really be nice.  Then when I am all done I can just upload it to github.

**Comment by ihnatenkoo** [comment]
VSC has extension "Live Server". Use it, and when you will save your index.hstml you can see changes in the browser

**Comment by hansemso** [comment]
Ah, ok, thanks.  I will look into it.
